<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>MiSMui—Documentation</title>
  <meta name="description" content="This is a minimal example of using the MiSMui Software package to write a book." />
  <meta name="generator" content="bookdown 0.37 and GitBook 2.6.7" />

  <meta property="og:title" content="MiSMui—Documentation" />
  <meta property="og:type" content="book" />
  
  <meta property="og:description" content="This is a minimal example of using the MiSMui Software package to write a book." />
  <meta name="github-repo" content="rstudio/bookdown-demo" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="MiSMui—Documentation" />
  
  <meta name="twitter:description" content="This is a minimal example of using the MiSMui Software package to write a book." />
  

<meta name="author" content="Tao Chen" />


<meta name="date" content="2024-02-27" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  

<link rel="next" href="hmm-build.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>



<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">A Minimal Book Example</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#background"><i class="fa fa-check"></i><b>1.1</b> Background</a></li>
<li class="chapter" data-level="1.2" data-path="index.html"><a href="index.html#about"><i class="fa fa-check"></i><b>1.2</b> About</a>
<ul>
<li class="chapter" data-level="1.2.1" data-path="index.html"><a href="index.html#relate-tools"><i class="fa fa-check"></i><b>1.2.1</b> Relate Tools</a></li>
<li class="chapter" data-level="1.2.2" data-path="index.html"><a href="index.html#relete-database"><i class="fa fa-check"></i><b>1.2.2</b> Relete Database</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="hmm-build.html"><a href="hmm-build.html"><i class="fa fa-check"></i><b>2</b> HMM Build</a>
<ul>
<li class="chapter" data-level="2.1" data-path="hmm-build.html"><a href="hmm-build.html#hmm-model-introduction"><i class="fa fa-check"></i><b>2.1</b> HMM Model Introduction</a></li>
<li class="chapter" data-level="2.2" data-path="hmm-build.html"><a href="hmm-build.html#hmm-model-build"><i class="fa fa-check"></i><b>2.2</b> HMM Model Build</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="hmm-build.html"><a href="hmm-build.html#local-protein-file-analysis"><i class="fa fa-check"></i><b>2.2.1</b> Local Protein File Analysis</a></li>
<li class="chapter" data-level="2.2.2" data-path="hmm-build.html"><a href="hmm-build.html#local-hmm-model-analysis"><i class="fa fa-check"></i><b>2.2.2</b> Local HMM Model Analysis</a></li>
<li class="chapter" data-level="2.2.3" data-path="hmm-build.html"><a href="hmm-build.html#public-database"><i class="fa fa-check"></i><b>2.2.3</b> Public Database</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="hmm-search.html"><a href="hmm-search.html"><i class="fa fa-check"></i><b>3</b> HMM Search</a>
<ul>
<li class="chapter" data-level="3.1" data-path="hmm-search.html"><a href="hmm-search.html#hmm-search-basic-analyis"><i class="fa fa-check"></i><b>3.1</b> HMM Search Basic Analyis</a></li>
<li class="chapter" data-level="3.2" data-path="hmm-search.html"><a href="hmm-search.html#hmm-search-expanded-analysis"><i class="fa fa-check"></i><b>3.2</b> HMM Search Expanded analysis</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="blast-1.html"><a href="blast-1.html"><i class="fa fa-check"></i><b>4</b> Blast</a>
<ul>
<li class="chapter" data-level="4.1" data-path="blast-1.html"><a href="blast-1.html#blast-introduction"><i class="fa fa-check"></i><b>4.1</b> Blast Introduction</a></li>
<li class="chapter" data-level="4.2" data-path="blast-1.html"><a href="blast-1.html#blast-2"><i class="fa fa-check"></i><b>4.2</b> Blast</a>
<ul>
<li class="chapter" data-level="4.2.1" data-path="blast-1.html"><a href="blast-1.html#blast-input"><i class="fa fa-check"></i><b>4.2.1</b> Blast Input</a></li>
<li class="chapter" data-level="4.2.2" data-path="blast-1.html"><a href="blast-1.html#blast-hmm-search-sequences-analysis"><i class="fa fa-check"></i><b>4.2.2</b> Blast-HMM Search Sequences Analysis</a></li>
<li class="chapter" data-level="4.2.3" data-path="blast-1.html"><a href="blast-1.html#blast-local-sequences-analysis"><i class="fa fa-check"></i><b>4.2.3</b> Blast-Local Sequences Analysis</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="phylogenetic-tree.html"><a href="phylogenetic-tree.html"><i class="fa fa-check"></i><b>5</b> Phylogenetic Tree</a>
<ul>
<li class="chapter" data-level="5.1" data-path="phylogenetic-tree.html"><a href="phylogenetic-tree.html#introduction-1"><i class="fa fa-check"></i><b>5.1</b> Introduction</a></li>
<li class="chapter" data-level="5.2" data-path="phylogenetic-tree.html"><a href="phylogenetic-tree.html#analysis"><i class="fa fa-check"></i><b>5.2</b> Analysis</a>
<ul>
<li class="chapter" data-level="5.2.1" data-path="phylogenetic-tree.html"><a href="phylogenetic-tree.html#local-fasta-data-analysis"><i class="fa fa-check"></i><b>5.2.1</b> Local Fasta Data Analysis</a></li>
<li class="chapter" data-level="5.2.2" data-path="phylogenetic-tree.html"><a href="phylogenetic-tree.html#local-tree-data-analysis"><i class="fa fa-check"></i><b>5.2.2</b> Local Tree Data Analysis</a></li>
<li class="chapter" data-level="5.2.3" data-path="phylogenetic-tree.html"><a href="phylogenetic-tree.html#hmm-search-filter-data-analysis"><i class="fa fa-check"></i><b>5.2.3</b> HMM Search Filter Data Analysis</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="antismash-1.html"><a href="antismash-1.html"><i class="fa fa-check"></i><b>6</b> Antismash</a>
<ul>
<li class="chapter" data-level="6.1" data-path="antismash-1.html"><a href="antismash-1.html#antismash-introduction"><i class="fa fa-check"></i><b>6.1</b> Antismash Introduction</a></li>
<li class="chapter" data-level="6.2" data-path="antismash-1.html"><a href="antismash-1.html#antismash-analysis"><i class="fa fa-check"></i><b>6.2</b> Antismash Analysis</a>
<ul>
<li class="chapter" data-level="6.2.1" data-path="antismash-1.html"><a href="antismash-1.html#mism-database"><i class="fa fa-check"></i><b>6.2.1</b> MiSM Database</a></li>
<li class="chapter" data-level="6.2.2" data-path="antismash-1.html"><a href="antismash-1.html#personal-data-analysis"><i class="fa fa-check"></i><b>6.2.2</b> Personal Data Analysis</a></li>
<li class="chapter" data-level="6.2.3" data-path="antismash-1.html"><a href="antismash-1.html#ncbi-accession-id"><i class="fa fa-check"></i><b>6.2.3</b> NCBI Accession ID</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7" data-path="lab-bacteria-map.html"><a href="lab-bacteria-map.html"><i class="fa fa-check"></i><b>7</b> Lab Bacteria Map</a>
<ul>
<li class="chapter" data-level="7.1" data-path="lab-bacteria-map.html"><a href="lab-bacteria-map.html#introduction-2"><i class="fa fa-check"></i><b>7.1</b> Introduction</a></li>
<li class="chapter" data-level="7.2" data-path="lab-bacteria-map.html"><a href="lab-bacteria-map.html#analysis-1"><i class="fa fa-check"></i><b>7.2</b> Analysis</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">MiSMui—Documentation</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="header">
<h1 class="title">MiSMui—Documentation</h1>
<p class="author"><em>Tao Chen</em></p>
<p class="date"><em>2024-02-27</em></p>
</div>
<div id="introduction" class="section level1 hasAnchor" number="1">
<h1><span class="header-section-number">Chapter 1</span> Introduction<a href="index.html#introduction" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<p><strong>MiSMui</strong> is an R shiny application that can be easily launched from a local web browser to help scientists without programming expertise to analyze (MiSM Database) microbial secondary metabolites biosynthesis gene cluster database.</p>
<div id="background" class="section level2 hasAnchor" number="1.1">
<h2><span class="header-section-number">1.1</span> Background<a href="index.html#background" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p><strong>The secondary metabolism</strong> of microbial flora has produced some bioactive compounds with potential medicinal value, including biosynthetic pathways of many chemicals, some of which have been used as antibiotics, cholesterol-lowering drugs or anti-tumor drugs. The biosynthetic genes responsible for encoding these secondary metabolites are usually gathered at a certain location in the chromosome. Such genes are called <strong>“secondary metabolites biosynthetic gene clusters”</strong> (<strong>BGC</strong>). Through the analysis of gene clusters in secondary metabolites in the genetic structure, they can biosynthesize secondary metabolites. Make a way to make predictions.</p>
<p>In order to explore biosynthesis more comprehensively, we have built a <strong>MiSM database</strong>. <strong>MiSM database</strong> can provide interested BGC-related information based on a large number of microbial flora, which can help researchers study the biosynthesis pathways of flora more comprehensively. The data sources of the <strong>MiSM database</strong> include the microbial flora and laboratory internal flora that have been reported in NCBI.At the same time, in order to better use the <strong>MiSM database</strong> , we have built an interactive analysis tool called <strong>MiSMui</strong> based on the Rshiny framework. <strong>MiSMui</strong> can search the <strong>MiSM database</strong> through the HMM model to identify similar sequences, and at the same time, it can perform Blast on protein sequences of interest to further analyze their similarities. In short, <strong>MiSMui</strong> is designed to serve the <strong>MiSM database</strong>.</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:unnamed-chunk-1"></span>
<img src="figure/Lab_MiSMui_Workflow.jpg" alt="MiSMui-Workflow" width="100%" />
<p class="caption">
Figure 1.1: MiSMui-Workflow
</p>
</div>
</div>
<div id="about" class="section level2 hasAnchor" number="1.2">
<h2><span class="header-section-number">1.2</span> About<a href="index.html#about" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<div id="relate-tools" class="section level3 hasAnchor" number="1.2.1">
<h3><span class="header-section-number">1.2.1</span> Relate Tools<a href="index.html#relate-tools" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="hmmer" class="section level4 hasAnchor" number="1.2.1.1">
<h4><span class="header-section-number">1.2.1.1</span> HMMer<a href="index.html#hmmer" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><strong>HMMER</strong> is used for searching sequence databases for sequence homologs, and for making sequence alignments. It implements methods using probabilistic models called profile hidden Markov models (profile HMMs).
<strong>HMMER</strong> is often used together with a profile database, such as Pfam or many of the databases that participate in Interpro. But <strong>HMMER</strong> can also work with query sequences, not just profiles, just like BLAST. For example, you can search a protein query sequence against a database with phmmer, or do an iterative search with jackhmmer.
<strong>HMMER</strong> is designed to detect remote homologs as sensitively as possible, relying on the strength of its underlying probability models. In the past, this strength came at significant computational expense, but as of the new HMMER3 project, HMMER is now essentially as fast as BLAST.
<strong>HMMER</strong> can be downloaded and installed as a command line tool on your own hardware, and now it is also more widely accessible to the scientific community via new search servers at the European Bioinformatics Institute.</p>
</div>
<div id="antismash" class="section level4 hasAnchor" number="1.2.1.2">
<h4><span class="header-section-number">1.2.1.2</span> Antismash<a href="index.html#antismash" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>The <strong>antiSMASH</strong> framework allows the detection of clusters of co-occurring biosynthesis genes in genomes, called Biosynthetic Gene Clusters (BGCs). BGCs often contain all the genes required for the biosynthesis of one or more Natural Products (NPs), also known as specialized or secondary metabolites. NPs show interesting biological activities and many of them have been developed into essential medicines, including antibiotics (penicillin, streptomycin), anti-cancer drugs (bleomycin, doxorubicin), or cholesterol-lowering agents (lovastatin). This makes NPs and their encoding BGCs highly relevant from both a commercial and scientific perspective.</p>
</div>
<div id="blast" class="section level4 hasAnchor" number="1.2.1.3">
<h4><span class="header-section-number">1.2.1.3</span> Blast+<a href="index.html#blast" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>The NCBI provides a suite of command-line tools to run <strong>BLAST</strong> called <strong>BLAST+</strong> . This allows users to perform <strong>BLAST</strong> searches on their own server without size, volume and database restrictions. <strong>BLAST+</strong> can be used with a command line so it can be integrated directly into your workflow.
The Basic Local Alignment Search Tool (<strong>BLAST</strong>) finds regions of local similarity between sequences. The program compares nucleotide or protein sequences to sequence databases and calculates the statistical significance of matches. <strong>BLAST</strong> can be used to infer functional and evolutionary relationships between sequences as well as help identify members of gene families.</p>
</div>
</div>
<div id="relete-database" class="section level3 hasAnchor" number="1.2.2">
<h3><span class="header-section-number">1.2.2</span> Relete Database<a href="index.html#relete-database" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="ncbi" class="section level4 hasAnchor" number="1.2.2.1">
<h4><span class="header-section-number">1.2.2.1</span> NCBI<a href="index.html#ncbi" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>The NCBI database is a collection of online resources that provide access to information on genomes, genes, proteins, chemicals, and other biological data. The NCBI database includes the following features:</p>
<p><strong>NCBI Genome:</strong> A resource that organizes information on genomes, including sequences, maps, chromosomes, assemblies, and annotations. Users can browse by organism, download data, or submit their own genome data.
<strong>NCBI Assembly:</strong> A resource that provides access to genome assemblies from various sources, such as GenBank, RefSeq, and T2T. Users can search, download, or compare genome assemblies.
<strong>NCBI BioProject:</strong> A resource that provides an overview of the biological data available for a given project. Users can search, browse, or submit BioProjects, which are collections of related data sets.
<strong>NCBI BioSample:</strong> A resource that provides information on the biological source materials used in genomic studies. Users can search, browse, or submit BioSamples, which are records of the origin, collection, and processing of biological specimens.
<strong>NCBI Genome Data Viewer:</strong> A resource that allows users to visualize and explore genomic data, such as annotations, variants, expression, and alignments. Users can upload their own data or use data from NCBI or other sources.
<strong>NCBI Datasets:</strong> A resource that provides easy access to genome sequence and annotation data. Users can download data sets for a given taxon, gene, or protein, or use the Datasets tools to analyze and compare genomic data.</p>
</div>
<div id="antismash-database" class="section level4 hasAnchor" number="1.2.2.2">
<h4><span class="header-section-number">1.2.2.2</span> Antismash Database<a href="index.html#antismash-database" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><strong>The antiSMASH database offers researchers four main pieces of funtionality. </strong>
It is possible to browse the database by phylogeny or secondary metabolite type, giving an exploratory overview of the avaliable biosynthetic gene cluster predictions. Selecting a cluster for closer inspection will redirect to the antiSMASH results page, offering all the usual annotations.
Alteratively, it is possible to query for specific biosynthetic gene cluster types, monomers and many more. The “build a query” page allows to construct complicated, detailed queries in an easy manner.
If you want to find BGC regions containing protein sequences similar to a protein you are interested in, or containing a RiPP core peptide similar to a core peptide you are interested in, the protein sequence search is the tool to used.
Lastly, a number of summary statistics are available to give a general idea of the types of secondary metabolite production represented by the database.</p>
</div>
<div id="mibig-database" class="section level4 hasAnchor" number="1.2.2.3">
<h4><span class="header-section-number">1.2.2.3</span> Mibig Database<a href="index.html#mibig-database" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><strong>The Minimum Information about a Biosynthetic Gene cluster (MIBiG)</strong> specification provides a robust community standard for annotations and metadata on biosynthetic gene clusters and their molecular products.
<strong>MIBiG</strong> is a Genomic Standards Consortium project that builds on the Minimum Information about any Sequence (MIxS) framework.
<strong>MIBiG</strong> will facilitate the standardized deposition and retrieval of biosynthetic gene cluster data as well as the development of comprehensive comparative analysis tools. It will empower next-generation research on the biosynthesis, chemistry and ecology of broad classes of societally relevant bioactive secondary metabolites, guided by robust experimental evidence and rich metadata components.
The JSON specifications of the <strong>MIBiG</strong> schema and options are freely available for download and can be used to validate <strong>MIBiG</strong> data from other sources. Also, a tarball with all <strong>MIBiG</strong> entries in raw JSON format is available for download. To get an idea of what kind of information is included in <strong>MIBiG</strong> entries, please have a look at the visualized entries for erythromycin, actinorhodin, calcium-dependent antibiotic, microbisporicin and marneral. They can be downloaded in JSON format by clicking the corresponding button (down-pointing arrow) at the top right of their page.</p>

</div>
</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>

<a href="hmm-build.html" class="navigation navigation-next navigation-unique" aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/USERNAME/REPO/edit/BRANCH/index.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["_main.pdf", "_main.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
